install.sh

```diff
@@ -733,6 +783,37 @@ service_enable_and_start() {
     return 0
 }
 
+# --- create symlink to kubeconfig
+setup_kubeconfig() {
+    DEFAULT_CONFIG=${HOME}/.kube/config
+    if [ ! -f "${KUBECONFIG}" ]; then
+        info "Waiting for kubeconfig at ${KUBECONFIG}..."
+        for _ in $(seq 1 20); do
+            [ -f "${KUBECONFIG}" ] && break
+            sleep 1
+        done
+        [ -f "${KUBECONFIG}" ] || warn "Kubeconfig does not yet exist at ${KUBECONFIG}"
+    fi
+
+    if [ -L ${DEFAULT_CONFIG} ]; then
+        CONFIG_LINK=$(readlink -f ${DEFAULT_CONFIG} || true)
+        if [ "${CONFIG_LINK}" == "${KUBECONFIG}" ] || echo ${CONFIG_LINK} | grep -vq k3s; then
+            info "Skipping kubeconfig symlink, ~/.kube/config link already exists to ${CONFIG_LINK}"
+            return
+        fi
+    elif [ -f ${DEFAULT_CONFIG} ]; then
+        info "Skipping kubeconfig symlink, ~/.kube/config file already exists"
+        return
+    fi
+
+    if [ ! -d $(dirname ${DEFAULT_CONFIG}) ]; then
+        mkdir -p $(dirname ${DEFAULT_CONFIG})
+    fi
+
+    ln -sf ${KUBECONFIG} ${DEFAULT_CONFIG}
```

Comment:
Without quotes here, eventual whitespaces in the file names/paths will break this.